src/seedu/addressbook/data/person/Person.java

```diff
@@ -19,6 +19,16 @@
     /**
      * Assumption: Every field must be present and not null.
      */
+
+
+    public String getPrintableString(Printable... printables){
+        String res = "";
+        for(Printable x: printables){
+            res = res+ x.getPrintableString();
```

Comment:
:bulb: *Java tip*: Java 8 has a `StringJoiner` class that can join multiple strings using a delimiter e.g. `aaa` `bbb` `ccc` --> `aaa, bbb, ccc` https://www.mkyong.com/java8/java-8-stringjoiner-example/